composer.json

```diff
@@ -29,16 +29,13 @@
 	"minimum-stability": "RC",
 	"scripts": {
 		"check-cs": [
-			"@php ./vendor/squizlabs/php_codesniffer/bin/phpcs"
+			"@php vendor/bin/phpcs"
 		],
 		"fix-cs": [
-			"@php ./vendor/squizlabs/php_codesniffer/bin/phpcbf"
-		],
-		"install-codestandards": [
```

Comment:
@jrfnl It appears that `install-codestandards` wasn't used or referenced anywhere in the documentation, which is why I assumed that it wasn't used.

The task of configuring the code standards for `phpcs` is handled by `dealerdirect/phpcodesniffer-composer-installer` and done automatically during `composer install`. That is also how we require and install the main `squizlabs/php_codesniffer` dependency. So I'm wondering how a developer could have `squizlabs/php_codesniffer` installed under `/vendor` without having run the `composer install`.

Reply:
That script is deprecated and was so annotated [in the PR which made it so](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/1591).